APIManager.js

```diff
@@ -0,0 +1,165 @@
+//This is the class that will manage all your APIs
+
+class APIManager
+{
+    constructor()
+    {
+        this.data = {};
+    }
+
+    loadData()
+    {
+        this.data.users = new UsersAPI().loadData();
+        this.data.quotes = new QuotesAPI().loadData();
+        this.data.bacons = new BaconsAPI().loadData();
+        this.data.pokemons = new PokemonsAPI().loadData();
+        return this.data;
+    }
+
+    // set data(data)
```

Comment:
make sure to delete all the comments before you committing your code a final commit